src/webviews/apps/plus/graph/graph.scss

```diff
@@ -372,6 +372,11 @@ button:not([disabled]),
 	vertical-align: middle;
 	line-height: 2rem;
 	letter-spacing: normal;
+
+	&.mini-icon {
+		font-size: 10px;
```

Comment:
Nit
```suggestion
		font-size: 1rem;
```

Reply:
Done in commit: [Uses rem for fontsize](https://github.com/gitkraken/vscode-gitlens/pull/2441/commits/1d080b7667a8ca5165c192f879ca90cabc5c30b3)